docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -3,47 +3,42 @@ sidebar_position: 2
 
 There are a few mechanisms that work differently to how they would on a centralised exchange, in order to keep the markets solvent. They include:
 - [**Margin**](#margin): When a party opens a position: the *initial margin* requirement is calculated automatically depending on the market's risk model. If the market moves against the party, and the margin towards the *maintenance level*, Vega will *search* for more collateral in the general account, to avoid liquidating the position. Margin can also be *released* if the position is in sufficient profit. Other positions in markets with the same settlement asset may also interact with the same general account. Therefore, Vega is a cross-margin based system by default.
-- [**Mark to market**](#mark-to-market): Mark to market on Vega happens much more frequently than on an exchange where counterparties are not pseudonymous. Every time a trade happens and moves the last traded price, the mark to market price is recalculated. Mark to market is used to move assets into your margin account (from someone else's) if you are in profit, or out of your margin account if not.
+- [**Mark to market**](#mark-to-market): Mark to market on Vega happens much more frequently than typical exchanges. Every time a trade happens and moves the last traded price, positions are marked to market. Marking to market is used to move assets into your margin account (from someone else's) if you are in profit, or out of your margin account if not.
 
 ## Mark to market
-Marking to market refers to settling gains and losses due to changes in the market value of the underlying product. Marking to market aims to provide a realistic appraisal of a position based on the current market conditions.
-
-When marking to market, the protocol takes the current market price and recalculates traders' margin requirements based on how their position is affected by price moves.
+Marking to market refers to settling gains and losses due to changes in the market value. Marking to market aims to provide a realistic appraisal of a position based on the current market conditions.
 
-If the market price goes up, a trader that holds a long position receives money in their margin account – equal to the underlying's change in value – from a trader that holds a short position, and conversely if the value goes down, the holder of the short position receives money from the holder of the long position.
+If the market price goes up, traders that hold a long positions receive money into their margin account – equal to the change in the notional value of their positions – from traders that hold short positions, and conversely if the value goes down, the holders of the short positions receive money from the holders of long positions.
```

Comment:
```suggestion
If the market price goes up, traders that hold a long position receive money into their margin account – equal to the change in the notional value of their positions – from traders that hold short positions, and conversely if the value goes down, the holders of the short positions receive money from the holders of long positions.
```

Reply:
Modified now to "trader that hold long positions". I wanted to indicate that it's not a 1 to 1 flow, we just collect money from all those who made a loss and distribute to among those that made a profit, but it's not like people are paired in any specific way.

Reply:
Sounds good - I should have commented on this one as what I want to fix is the plural/singular agreement, but I don't mind how it's done, as long as they agree.